docs/data/introduction/priority-support/priority-support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: MUI X - Priority Support
+---
+
+# Priority Support [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+<p class="description">Learn how to use priority technical support for MUI components.</p>
+
+## What is priority support?
+
+Priority Support is a premium service ensuring fast, expert assistance for your critical applications, with expedited troubleshooting and priority handling of inquiries.
+
+Priority support takes precedence over standard support, which is included by default in paid plans. It is backed by an SLA, whereas standard support is not. Support requests are prioritized in the following order:
```

Comment:
> It is backed by an SLA, whereas standard support is not

Technically, we must also describe the service level of the standard support, which we do on https://mui.com/legal/technical-support-sla/

```suggestion
Priority support takes precedence over standard support, which is included by default in paid plans. Support requests are prioritized in the following order:
```